File: leetcode/0000-0099/75. Sort Colors/75.cpp
class Solution {
public:
    void sortColors(vector<int>& nums) {
        vector<int> counts = {0, 0, 0};
        for (int n : nums) {
            counts[n] += 1;
        }
        int c = 0;
        for (int i = 0; i < counts.size(); i++) {
            for (int j = 0; j < counts[i]; j++) {
                nums[c] = i;
                c++;
            }
        }
    }
};

File: leetcode/0000-0099/75. Sort Colors/75.py
class Solution:
    def sortColors(self, nums: List[int]) -> None:
        """
        Do not return anything, modify nums in-place instead.
        """

        counts = [0, 0, 0]

        for n in nums:
            counts[n] += 1

        i = 0
        for n in range(len(counts)):
            for j in range(counts[n]):
                nums[i] = n
                i += 1
        return nums
# TC:O(n) SC:O(c)